src/main/java/edu/unc/lib/boxc/migration/cdm/status/DescriptionsStatusService.java

```diff
@@ -62,7 +62,7 @@ public void reportStats(int totalObjects, Verbosity verbosity) {
                 showFieldListValues(indexedIds);
             }
         } catch (IOException e) {
-            outputLogger.info("Failed to list MODS records: {}", e.getMessage());
+            outputLogger.info("Failed to list MODS records: {}", e);
```

Comment:
Might make sense to add in a regular logger line here and just send the message to the outputLogger